android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -44,11 +48,12 @@ open class Node<V : RibView>(
         internal const val KEY_ROUTER = "node.router"
         internal const val KEY_INTERACTOR = "node.interactor"
         internal const val KEY_VIEW_STATE = "view.state"
-        private val requestCodeRegistry = RequestCodeRegistry(8)
+        private val requestCodeRegistry = RequestCodeRegistry()
     }
 
     init {
         router.node = this
+        interactor.node = this
```

Comment:
In general, I see the loop (well know from MVP ages, Presenter-View) there.
Is it not one of the reasons why we don't like MVP?

Introducing Node entity we just jump back to the same issue.
We have to discuss, maybe I am a drama queen.
@lukaville @ShikaSD

Reply:
Node here is doing this stuff under the hood, and client code never knows anything about this mechanism. I don't see an easy way to escape the circular dependency though.

Reply:
@jeksor agree with Zsolt, it's not exposed to the clients so for me it's OK

Reply:
@lukaville  +1

Reply:
I'm leaving it then.